parser/sqlparser/sql.y

```diff
@@ -613,6 +635,42 @@ create_statement:
     $$ = &DBDDL{Action: CreateStr, DBName: string($4)}
   }
 
+option_name:
+ table_id
```

Comment:
This looks like a hack, what was the problem with using just an identifier or something like that? (check out the table valued function argument names)